### PR TITLE
Simple redirects

### PR DESCRIPTION
**Pull request by tlienart**
```
{{redirect some/path.html}}
```

will add `[site]/some/path.html` and link to the page that has the hfun.

assumption: `some/path.html` does not clash (if it doesn't, whatever exists will stay as is and the redirect will not be created).

### CODE REVIEW

File: src/converter/html/functions.jl [hunk 1]
```diff
@@ -219,3 +219,39 @@ function hfun_taglist()::String
...
+        dst = joinpath(path(:site), addr)
+    end
+    isfile(dst) && return ""
+    mkpath(splitdir(dst)[1])
+    write(dst, """
+    <!-- Generated Redirect -->
+    <!doctype html>
+    <html>
+    <head>
+      <meta http-equiv="refresh" content="0; url=$(locvar(:fd_url))">
```
Comment (shashi) [inline src/converter/html/functions.jl:252]: should this prefix with `prepath`? it currently does not...

Comment (tlienart) [reply src/converter/html/functions.jl:252]: ah, crap, so the error is not here but there: https://github.com/tlienart/Franklin.jl/blob/d44d6d0d76cf9656e483146d71f52f8dd2f37448/src/converter/html/link_fixer.jl#L82 the regex should include `url` 

I'll fix this asap; thanks for the heads up

Comment (tlienart) [reply src/converter/html/functions.jl:252]: Fixed in the (pending) release 0.9.16

Comment (shashi) [reply src/converter/html/functions.jl:252]: Thank you!

Comment (shashi) [reply src/converter/html/functions.jl:252]: I see how it's done. Cool!

**Pull request merged**